### PR TITLE
Visualization updates

### LINKED ISSUE #19: Test visualize
Increase code coverage on visualize.py by testing the value errors are raised, see codecov report for details.

### PR DESCRIPTION
**Pull request by zofiakaczmarek**
Fixes #19  .

## Description of the Change

Added new visualization tests to cover different possible cases for input dimensions in visualization.py.
(+ small animation update)

## Checklist

Please check all that apply to your proposed changes

- [ ] Added package dependency
- [ ] Added data
- [x] Documentation change
- [x] code change

### CODE REVIEW

**Review by astrophpeter — COMMENTED** [review]
test raise execption with pytest and not unittest please

File: tests/test_visualize.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 import numpy as np
 from popclass.model import PopulationModel
 from popclass.visualization import plot_population_model
+from unittest import TestCase
```
[the remark this answers is not in the document]
Comment (zofiakaczmarek) [reply tests/test_visualize.py]: No, tricky cases are great :)
Thanks, now replaced!

**Comment by astrophpeter** [comment]
great, looks good to me! will merge shortly

**Comment by zofiakaczmarek** [comment]
Great!
I see there is still no test coverage for plotting samples - might add another test soon-ish to cover that as well.

Best,
Zofia

On Friday, September 06, 2024 16:38 CEST, Peter McGill ***@***.***> wrote:

 
 
great, looks good to me! will merge shortly
—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***>
[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/LLNL/popclass/pull/30#issuecomment-2334208050", "url": "https://github.com/LLNL/popclass/pull/30#issuecomment-2334208050", "name": "View Pull Request" }, "description": "View this Pull Request on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Pull request merged**